From 95455453b11d866027873cb72f276e10680a2cf1 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Sat, 26 Jan 2019 20:14:30 +0100
Subject: [PATCH] Fix compilation for lack of docbook2x-man

The guard in doc/Makefile.am was supposed to protect against
publishing release tarballs lacking a compiled xmlwf.1 man page
without noticing.  Now it does no more than advertised.
---
 expat/Changes         |    1 +
 expat/doc/Makefile.am |    5 ++++-
 2 files changed, 5 insertions(+), 1 deletions(-)

Signed-off-by: Javier Campos <jcampos@mvista.com>

Index: expat-2.2.6/doc/Makefile.am
===================================================================
--- expat-2.2.6.orig/doc/Makefile.am
+++ expat-2.2.6/doc/Makefile.am
@@ -28,14 +28,17 @@
 # OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 # USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+.PHONY: dist-hook  # not inside conditional to avoid automake warning
+
+if WITH_DOCBOOK
 dist_man_MANS = xmlwf.1
 
 xmlwf.1: xmlwf.xml
-if WITH_DOCBOOK
 	-rm -f $@
 	$(DOCBOOK_TO_MAN) $<
 	test -f $@ || mv XMLWF.1 $@
 else
+dist-hook:
 	@echo 'ERROR: Configure with --with-docbook for "make dist".' 1>&2
 	@false
 endif
